KsApi/GraphMutation.swift

```diff
@@ -10,7 +10,7 @@ protocol GraphMutation: CustomStringConvertible {
   var input: Input { get }
 }
 
-public struct GraphMutationEmptyResponseEnvelope: Decodable {}
+public struct EmptyResponseEnvelope: Decodable {}
```

Comment:
Removing the `GraphMutation` prefix from this like we did in the previous PR for the WatchProject mutation.